Bottom: 8dd330d1501c9486747af032c0cb1af84a8f48eb
Top:    4d41f56b2d3489a7e07f992a783a2ceef9436137
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-04-25 15:24:08 -0500

Cleaner method to get ip address


---
diff --git a/docker/docker-exim4/docker-entrypoint.sh b/docker/docker-exim4/docker-entrypoint.sh
index c629055..a99acea 100755
--- a/docker/docker-exim4/docker-entrypoint.sh
+++ b/docker/docker-exim4/docker-entrypoint.sh
@@ -29,7 +29,7 @@ case ${1} in
                 && rm -rf /etc/mailname
             echo "${HOSTNAME}" > /etc/mailname
             # set local IP addr
-            LOCAL_IP_ADDR=$(grep "[ \t]*${HOSTNAME}[ \t]*" /etc/hosts|cut -f 1)
+            LOCAL_IP_ADDR=$(getent hosts $HOSTNAME | cut -d ' ' -f 1)
             sed -i 's|dc_local_interfaces=.*|dc_local_interfaces='"'${LOCAL_IP_ADDR}'"'|' ${CONFIG_PATH}/update-exim4.conf.conf
             # copy certificate
             [ -e /etc/ssl/private/exim.crt ] \
